src/frontend/src/components/Landing/index.tsx

```diff
@@ -70,113 +76,369 @@ class Landing extends React.Component {
+                      <div className="bl bw2 f5 b--blue pb3 ph3 ml3 ml4-ns" >
+                        Here I am! I am the buried treasure!
+                      </div></DisclosurePanel>
+                   </Disclosure> </li>
+                    
+                    <li className="bt bw2 b--lightest-blue1">
+                    <Disclosure>
+                      <DisclosureButton className="flex-ns w-100 relative navy hover-blue pv3 ph3 ph4-ns">
+                        <span className="w-70 db fw7 pr3 mb2 mb0-ns">
+                          Qiu-qiu Law
+                        </span>
+                        <span className="w-30 pr3">
+                          Portland
+                        </span>
+                        <span className="absolute top-0 right-0 pt3 ph3">
+                          <span aria-hidden="true" className="fas fa-angle-up"></span>
+                        </span>
+                      </DisclosureButton>
+                        <DisclosurePanel>
+                      <div className="bl bw2 f5 b--blue pb3 ph3 ml3 ml4-ns">
+                        <ul className="list mb3">
+                          <li className="flex-ns mb3">
+                            <span className="w4-ns db fw6 mr3">Analysis Cost</span>
+                            <span>Free</span>
+                          </li>
+                          <li className="flex-ns mb3">
+                            <span className="w4-ns db fw6 mr3">Filing Cost</span>
+                            <span>NA</span>
+                          </li>
+                          <li className="flex-ns mb3">
+                            <span className="w4-ns db fw6 mr3">Court Fees</span>
+                            <span>Not Included</span>
+                          </li>
+                          <li className="flex-ns mb3">
+                            <span className="w4-ns db fw6 mr3">Income Restrictions</span>
+                            <span>Under 45k Annual</span>
+                          </li>
+                          <li className="flex-ns mb3">
+                            <span className="w4-ns db fw6 mr3">Locations</span>
+                            <span>Portland Metro</span>
+                          </li>
+                        </ul>
+                        <p className="mw6 bt b--black-10 pv3">
+                          Send an email with your full name, previous names (including aliases and maiden names), and date of birth.
+                        </p>
+                        <ul className="list mb3">
+                          <li className="fw6 mb3">
+                            email@domain.com
+                          </li>
+                          <li className="fw6 mb2">
+                            555-555-5555
+                          </li>
+                        </ul>
+                          </div>
+                          </DisclosurePanel></Disclosure></li>
```

Comment:
@Ark20 I recommend pulling all this code into a new `<PartnerInfo>` component so we can just define all the styling just once. The `<li></li>` items also have shared style so we can just define that once and map it to all the partner details.